src/com/github/andlyticsproject/BaseActivity.java

```diff
@@ -99,7 +99,7 @@ public void startActivity(Class<?> clazz, boolean disableAnimation, boolean skip
 			intent.setFlags(Intent.FLAG_ACTIVITY_NO_ANIMATION);
 		}
 		if (skipDataReload) {
-			setSkipMainReload(true);
+			this.skipMainReload = true;
```

Comment:
actually `this.` isn't needed

Reply:
* https://github.com/AndlyticsProject/andlytics/pull/768/commits/19b9d6ec9369f27582ea0ad363e53ee9edd052b2#diff-591237715a12453bb434ffc23b215d9fR123 also calls the setter
* with both parts removed maybe even the setter method could be removed (haven't checked further possible usages of the method).

Reply:
Hi @AndyScherzinger and thank you for your reply. I only applied a single example of MIM, I know that there might be some other instances, but I didn't want to apply too many changes at once.
Yes, whenever all references of setter are remove even the method itself could be remove, but currently aDoctor doesn't support this feature.